### PR TITLE
Grammatical Tweaks (Section 08)

### PR DESCRIPTION
**Pull request by furey**
Following #336, this PR applies numerous grammatical tweaks to section 08 of the documentation.

### CODE REVIEW

File: 08-Lucid-ORM/01-Getting-Started.adoc [hunk 1]
```diff
@@ -8,14 +8,19 @@ category: lucid-orm
 
 toc::[]
 
-Lucid is an implementation of link:https://en.wikipedia.org/wiki/Active_record_pattern[Active Record, window="_blank"] pattern in JavaScript. If you are coming from the *Laravel* or the *Rails* world, then you may be quite familiar with it.
+*Lucid* is the AdonisJS implementation of the link:https://en.wikipedia.org/wiki/Active_record_pattern[active record pattern, window="_blank"].
+
+If you're familiar with the *Laravel* or *Ruby on Rails*, you'll find many similarities between Lucid and Laravel's link:https://laravel.com/docs/eloquent[Eloquent, window="_blank"] or Rails' link:https://guides.rubyonrails.org/active_record_basics.html[Active Record, window="_blank"].
```
Comment (RomainLanz) [inline 08-Lucid-ORM/01-Getting-Started.adoc]:
```suggestion
If you're familiar with *Laravel* or *Ruby on Rails*, you'll find many similarities between Lucid and Laravel's link:https://laravel.com/docs/eloquent[Eloquent, window="_blank"] or Rails' link:https://guides.rubyonrails.org/active_record_basics.html[Active Record, window="_blank"].
```

File: 08-Lucid-ORM/01-Getting-Started.adoc [hunk 2]
```diff
@@ -424,40 +466,41 @@ const users = await User.all()
...
 ----
 const User = use('App/Models/User')
 const user = await User.create(props)
 // user.serviceToken === undefined
+
 await user.reload()
 // user.serviceToken === 'E1Fbl3sjH'
 ----
 
+A model with properties set during a creation hook will require *reloading* to retreive the values set during that hook.
```
Comment (RomainLanz) [inline 08-Lucid-ORM/01-Getting-Started.adoc]:
```suggestion
NOTE: A model with properties set during a creation hook will require *reloading* to retreive the values set during that hook.
```

**Review by RomainLanz — CHANGES_REQUESTED** [review]

**Comment by RomainLanz** [comment]
I'm loving those PR 💜

**Comment by RomainLanz** [comment]
Thanks!

**Comment by furey** [comment]
@RomainLanz No worries! Wish I had more time to work through it all, but whenever I get the chance I keep chipping away at it, paragraph by paragraph. Cheers!

**Pull request merged**